src/main/resources/assets/styles/wizard/content-wizard-panel.less

```diff
@@ -62,9 +55,6 @@
     &._0-240,
     &._240-360,
     &._360-540 {
-      .wizard-header {
-        max-width: calc(100% - 105px);
-      }
```

Comment:
This piece of code used to fix this issue:

![image](https://user-images.githubusercontent.com/11612490/108871500-82669700-75f9-11eb-80b1-f601217f2902.png)